src/hotspot/share/classfile/verifier.cpp

```diff
@@ -277,14 +279,6 @@ bool Verifier::is_eligible_for_verification(InstanceKlass* klass, bool should_ve
   Symbol* name = klass->name();
 
   return (should_verify_class &&
-    // return if the class is a bootstrapping class
```

Comment:
This method should have a comment explaining what the eligibility criteria is and how/why `should_verify_class` should be set in different conditions.

Reply:
I moved the function near should_be_verified, so it would be clear what this does and why, and added a comment.